grafana/dashboard-template.json

```diff
@@ -684,27 +718,27 @@
         {
           "datasource": "${DS_PROMETHEUS}",
           "editorMode": "code",
-          "expr": "kyuubi_memory_usage_pools_PS_Eden_Space_used{instance=~\"$instance\",job=~\"$job\"}",
+          "expr": "kyuubi_memory_usage_pools_PS_Eden_Space_used{instance=~\"$instance\",$baseFilter}",
           "hide": false,
-          "legendFormat": "Eden Space Used",
+          "legendFormat": "${baseLegend} Eden Space Used",
           "range": true,
           "refId": "B"
         },
         {
           "datasource": "${DS_PROMETHEUS}",
           "editorMode": "code",
-          "expr": "sum(kyuubi_memory_usage_pools_Par_Eden_Space_committed)",
+          "expr": "kyuubi_memory_usage_pools_Par_Eden_Space_committed{instance=~\"$instance\",$baseFilter}",
           "hide": false,
-          "legendFormat": "Eden Space Committed",
+          "legendFormat": "${baseLegend} Eden Space Committed",
           "range": true,
           "refId": "A"
         },
         {
           "datasource": "${DS_PROMETHEUS}",
           "editorMode": "code",
-          "expr": "sum(kyuubi_memory_usage_pools_Par_Eden_Space_max)",
```

Comment:
not sure whether the `sum` is necessary, but I think we can remove such kind of metrics.

Because seems it does not exists for my kyuubi server + JDK17.

Reply:
I think only reserve the Memery Usage  + GC count is enough.

cc @pan3793